com/com/laboon/WorldTest.java

```diff
@@ -0,0 +1,47 @@
+package com.laboon;
+
+import static org.junit.Assert.*;
+
+import org.junit.*;
+
+public class WorldTest {
+	
+	//Tests what toString returns when you create an empty world
+	@Test
+	public void EmptyWorldTest(){
+		World emptyWorld = new World(0, 3, 25);
+		assertEquals(emptyWorld.toString(), "  \n");
+	}
+	
+	//Tests a 10 x 10 world for correctly sized string representation
```

Comment:
All of these tests are only checking for size, not actual results of the string.  If the new method showed all cells as alive, for example, it wouldn't fail.